tests/backup/backup_orphaned_delete_test.go

```diff
@@ -0,0 +1,362 @@
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts := ScheduleApplications(taskName)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+				scheduledAppContexts = append(scheduledAppContexts, ctx)
+			}
+		}
+		log.InfoD("Created namespaces %v", bkpNamespaces)
+	})
+
+	It("Delete backups of non admin user from px-admin", func() {
+		Step("Validate applications", func() {
+			log.InfoD("Validate applications")
+			ValidateApplications(scheduledAppContexts)
+		})
+
+		Step("Create a non-admin users to create the backups and restore", func() {
+			log.InfoD(fmt.Sprintf("Create a non-admin users to create the backups and restore"))
+			roles := [3]backup.PxBackupRole{backup.ApplicationOwner, backup.InfrastructureOwner, backup.ApplicationUser}
+			for _, role := range roles {
+				userName = fmt.Sprintf("testauser-%v-%s", role, RandomString(4))
+				firstName := fmt.Sprintf("FirstName-%v-%s", role, RandomString(4))
+				lastName := fmt.Sprintf("LastName-%v-%s", role, RandomString(4))
+				email := fmt.Sprintf("testuser-%v-%s@cnbu.com", role, RandomString(4))
+				err := backup.AddUser(userName, firstName, lastName, email, commonPassword)
+				log.FailOnError(err, "Failed to create user - %s", userName)
+				err = backup.AddRoleToUser(userName, role, fmt.Sprintf("Adding %v role to %s", role, userName))
+				log.FailOnError(err, "Failed to add role for user - %s", userName)
+				if role == backup.ApplicationOwner {
+					appAdminUserName = userName
+				} else if role == backup.InfrastructureOwner {
+					infraAdminUserName = userName
+				} else {
+					appUserName = userName
+				}
+				userNames = append(userNames, userName)
+				userUID, err := backup.FetchIDOfUser(userName)
+				log.FailOnError(err, "Failed to fetch uid for - %s", userName)
+				userIdMap[userName] = userUID
+			}
+		})
+
+		Step(fmt.Sprintf("Adding Credentials from px-admin user and sharing with required users"), func() {
+			log.InfoD(fmt.Sprintf("Adding Credentials from px-admin user and sharing with required users"))
+			adminCtx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-admin ctx")
+			for _, provider := range providers {
+				adminCloudCredUID = uuid.New()
+				adminCredName = fmt.Sprintf("autogenerated-cred-%v", time.Now().Unix())
+				err = CreateCloudCredential(provider, adminCredName, adminCloudCredUID, orgID, adminCtx)
+				log.FailOnError(err, "Failed to create cloud credential - %s", err)
+				log.Infof("Updating cloud credential ownership for non-admin users")
+				if provider != drivers.ProviderNfs {
+					err = UpdateCloudCredentialOwnership(adminCredName, adminCloudCredUID, []string{appAdminUserName}, nil, Read, Invalid, adminCtx, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying updation of ownership for cloudcredential- %s", adminCredName))
```

Comment:
How about including username in the log message?

dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying ownership update for cloud credential %s to user %s", adminCredName, []string{appAdminUserName}))